content/en/blog/_posts/2020-12-02-dockershim-faq.md

```diff
@@ -3,15 +3,17 @@ layout: blog
 title: "Dockershim Deprecation FAQ"
 date: 2020-12-02
 slug: dockershim-faq
-aliases: [ '/dockershim' ]
 ---
 
-This document goes over some frequently asked questions regarding the Dockershim
-deprecation announced as a part of the Kubernetes v1.20 release. For more detail
-on the deprecation of Docker as a container runtime for Kubernetes kubelets, and
-what that means, check out the blog post
+This article goes over some frequently asked questions regarding the Dockershim
+deprecation [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
+as a part of the Kubernetes v1.20 release. For more detail on the deprecation of Docker as a container
+runtime for Kubernetes kubelets, and what that means, check out the blog post
```

Comment:
Could we add the blogpost that was released a couple of months back, as well? 

Link: https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/

I feel like we could benefit from mentioning both of them.

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/ isn't a blog article, but yes the FAQ can link to it.

I suggest separating that change into its own PR. @divya-mohan0209 would you be willing to log an issue about making that change?

Reply:
Ah sorry, phrasing mistake @sftim 😂 . But yes, will do it! Only add the link to the FAQ doc right?

Reply:
Let's use that new issue to discuss exactly what we'd like to do.

Reply:
Logged an issue regarding this.